lesson06/script.js

```diff
@@ -1,9 +1,9 @@
-const lists = [
-    {to: "bookmark.html", img: "1.png", alt:"画像1", text: "ブックマーク"},
-    {to: "message.html", img: "2.png", alt:"画像2", text: "メッセージ"}
-];
-
 const promise = new Promise(resolve => {
+    const lists = [
+        {to: "bookmark.html", img: "1.png", alt:"画像1", text: "ブックマーク"},
+        {to: "message.html", img: "2.png", alt:"画像2", text: "メッセージ"}
+    ];
+    
     setTimeout(() => resolve(lists),3000);
 });
 
```

Comment:
I put a variable declaration in a Promise.
This is to make the code easier to read.